File: itsm/src/main/kotlin/co/brainz/itsm/serviceRequest/ServiceRequestController.java
package co.brainz.itsm.serviceRequest;

import java.util.List;

import javax.annotation.Resource;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.GetMapping;

import co.brainz.framework.sample.logging.controller.LoggingController;
import co.brainz.workflow.process.ProcessHist;

@Controller
public class ServiceRequestController {
	private static Logger logger = LoggerFactory.getLogger(ServiceRequestController.class);
	
    @Resource(name = "serviceRequestService")
    private ServiceRequestService serviceRequestService;
    
    @GetMapping("/processList")
    public String serviceRequest(Model model) {       
        serviceRequestService.setData();
        
        List<ProcessHist> processList = serviceRequestService.getProcessList();
        
        model.addAttribute("processList",processList);
        
        return "common/layout";
    }    
    
    @GetMapping("/writeCustomerRequest")
    public String writeCustomerRequest(Model model) {        
        
        serviceRequestService.setData();
        
        List<ProcessHist> processList = serviceRequestService.getProcessList();
        
        model.addAttribute("processList",processList);
        
        return "itsm/writeCustomerRequest";
    }   
}

File: itsm/src/main/kotlin/co/brainz/itsm/serviceRequest/ServiceRequestController.kt
package co.brainz.itsm.serviceRequest


import javax.annotation.Resource;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.GetMapping;
import co.brainz.framework.sample.logging.controller.LoggingController;
import co.brainz.workflow.process.ProcessHist;

@Controller
class ServiceRequestController {
	
	companion object {
	private val logger = LoggerFactory.getLogger(ServiceRequestController::class.java)
	}
	
	fun Logging() : Unit{
		logger.info("INFO{ }","ServiceRequestController")
	}
	
	@Resource(name = "serviceRequestService")
    private lateinit var serviceRequestService : ServiceRequestService
	
	@GetMapping("/processList")
    public fun serviceRequest(model : Model) : String {
		
	    serviceRequestService.setData();
		
		val processList : MutableList<ProcessHist> = serviceRequestService.getProcessList();
		 
		model.addAttribute("processList",processList);
		
		return "common/layout"	
	}
	
	@GetMapping("/writeCustomerRequest")
    public fun writeCustomerRequest(model : Model) : String {
		
		serviceRequestService.setData();
		
		val processList : MutableList<ProcessHist> = serviceRequestService.getProcessList()
		
		model.addAttribute("processList",processList);
		
		return "itsm/writeCutomerRequest"
	}
}

